Examples/UIExplorer/NavigationExperimental/NavigationAnimatedExample.js

```diff
@@ -99,8 +100,10 @@ class NavigationAnimatedExample extends React.Component {
   _renderHeader(/*NavigationSceneRendererProps*/ props) {
     return (
       <NavigationHeader
-        {...props}
-        getTitle={state => state.key}
+        navigationProps={props}
```

Comment:
I'm sorry but I'm not convinced. 

You can still split the props at the `render` function of `NavigationHeader` and pass them to `renderLeftComponent` and `renderRightComponent` separately.

The contract is that the `props` enforces the type `NavigationSceneRendererProps` and people would be able to build different kind of navigation header without inventing new props definitions.

For instance, what would the props like if you were doing this for NavigationBreadcrumbHeader? In that case, the header is literally just a component that contains a list of scenes (breadcrumbs) and the API may look like this:

```
<NavigationBreadcrumbHeader
  {...headerProps /*NavigationSceneRendererProps*/ }
  renderBreadCrumb={(breadCrumbProps /*NavigationSceneRendererProps*/) => {
    // render one bread crumb.....
  }
/> 
```

Having consistent type for rendering will make it easier to reuse views or swap views.